interfaces/seccomp/backend.go

```diff
@@ -58,35 +57,39 @@ var (
 	releaseInfoId            = release.ReleaseInfo.ID
 	releaseInfoVersionId     = release.ReleaseInfo.VersionID
 	requiresSocketcall       = requiresSocketcallImpl
-	seccompCompilerLookup    = seccompToBpfPath
+
+	snapSeccompVersionInfo = snapSeccompVersionInfoImpl
+	seccompCompilerLookup  = snapSeccompPath
 )
 
-func seccompToBpfPath(compiler string) (string, error) {
+func snapSeccompVersionInfoImpl(c Compiler) (string, error) {
+	return c.VersionInfo()
+}
+
+func snapSeccompPath(compiler string) (string, error) {
 	// FIXME: use cmd.InternalToolPath here once:
```

Comment:
I think this comment has to go. In light of the revelation about bootstrap phase it no longer applies.